card-game-client/app/components/ui/admin-card.tsx

```diff
@@ -167,23 +167,31 @@ export default function AdminCard(categoryName: AdminCardProps) {
                                     {filteredCardData.id}
                                 </TableCell>
                                 <TableCell>
-                                    <Image
-                                        src={
-                                            imageError[filteredCardData.id]
-                                                ? defaultImageUrl
-                                                : "http://ec2-54-176-67-195.us-west-1.compute.amazonaws.com:8080/uploadFiles/" +
-                                                  (filteredCardData.file
-                                                      ?.savedName || "")
-                                        }
-                                        alt={filteredCardData.subCategory || ""}
-                                        width={50}
-                                        height={50}
-                                        onError={() =>
-                                            handleImageError(
-                                                filteredCardData.id,
-                                            )
-                                        }
-                                    />
+                                    {filteredCardData.file?.extension &&
+                                     videoFileExtensions.includes(filteredCardData.file.extension) ? (
+                                        <video
+                                            src={fileUploadUrl + filteredCardData.file.savedName}
+                                            controls={false}
+                                            width={50}
+                                            height={50}
+                                        />
+                                    ) : (
+                                        <Image
+                                            src={
+                                                imageError[filteredCardData.id]
+                                                    ? defaultImageUrl
+                                                    : fileUploadUrl + (filteredCardData.file?.savedName || "")
+                                            }
```

Comment:
I'm assuming this fixes - The dashboard is setting the images on the preview cards to defaultImage, even when there is an image associated with the card, the only exception it seems like is the very last card

Reply:
It doesn't, the issue only occurs on the dashboard page for some reason. What happens is when I load the page i see the photos that were saved onto storage for a split second, then they immediately turn to the defaultImage (at least thats what the inspect tool on browser tells me).